UKHomeOffice/sgmr-service
# Accessibility updates

## Description
Add descriptive links for screen readers to make them accessible until we can update tables more extensively

## To Test
- turn on voice over
- go to manage reports
- tab to voyages
- > Links should read out the vessel, departure date, port, and arrival port
- go to people
- tab to a person
- > Link should read out `edit person firsname lastname`
- go to vessel
- tab to a vessel
- > Link should read out `edit vesselType named vesselName`


## Developer Checklist

\* Required






Ensure you delete the branch once the Pull Request is merged.
